src/lib/sql/sql_pipeline.cpp

```diff
@@ -32,13 +32,15 @@ SQLPipeline::SQLPipeline(const std::string& sql, std::shared_ptr<TransactionCont
   DebugAssert(parse_result.size() > 0, "Cannot create empty SQLPipeline.");
   _sql_pipeline_statements.reserve(parse_result.size());
 
-  bool seen_altering_statement = false;
+  auto seen_altering_statement = false;
+  auto sql_string_offset = 0u;
```

Comment:
This could use a comment on what it is used for.  The comment should explain, why we bother dealing with the SQL string at all